pkgs/love2d/example/assets/love_example_browser/main.lua

```
@@ -0,0 +1,196 @@
+
+function exf.clear()
+	love.graphics.setBackgroundColor(0, 0, 0)
+	love.graphics.setColor(1, 1, 1)
+	love.graphics.setLineWidth(1)
+	love.graphics.setLineStyle("smooth")
+	love.graphics.setBlendMode("alpha")
+	love.mouse.setVisible(true)
+end
+
+function exf.resume()
+	load = nil
```

Luacheck (1.2.0)

[warning] setting read-only global variable 'load'

(W121)